crates/duty-tracker/src/contract_manager.rs

```diff
@@ -1512,11 +1512,9 @@ impl ContractManagerCtx {
 
 async fn execute_duty(
     cfg: Arc<ExecutionConfig>,
-    output_handles: Arc<OutputHandles>,
+    outs: Arc<OutputHandles>,
```

Comment:
I think that we can also change here to be an `&OutputHandles` and not force the `Arc` in the parameter type. Let the `DeRef` do its job (unless the `execute_duty` uses `Arc::*` functions, which I think it doesn't, but I might be wrong).

Reply:
We can't do this because we need to be able to share the reference across threads which means you have to borrow for static. We can't do this because the fact that the event loop can end makes ctx drop which would cause a dropped while borrowed error.

Reply:
Ok!